test/unit/forge/routes/auth/oauth_spec.js

```diff
@@ -61,7 +61,7 @@ describe('OAuth', async function () {
             authURL.search = new URLSearchParams(params)
             return {
                 verifier,
-                authURL
+                authURL: `${authURL.pathname}${authURL.search}`
```

Comment:
```suggestion
                authURL: authURL.href
```

This could be a bit tidier, i think the .href would contain both pathname and search params (if you're not trying to exclude other things)

Reply:
Don't want to include the `https://localhost:3000` bit that gets included in href.